src/Publication/Publication.queries.js

```diff
@@ -3,6 +3,8 @@ import { SQL, raw } from '../database'
 
 export const PublicationQueries = Object.freeze({
   whereisActive: () => SQL`expires_at >= EXTRACT(epoch from now()) * 1000`,
+  andHasStatus: status =>
+    status != null ? SQL`AND status = ${status}` : SQL``,
```

Comment:
If we go with my suggestion of keeping this without the `AND` we can use ` 1 = 1 ` as the default value if status is null

Reply:
sabe